fdbserver/storageserver.actor.cpp

```diff
@@ -665,6 +667,34 @@ void validate(StorageServer* data, bool force = false) {
 }
 #pragma endregion
 
+void
+updateProcessStats(StorageServer* self)
+{
```

Comment:
We are going to be better off just getting the cpuUsage and diskUsage directly from the calls to platform.h instead of parsing the results from ProcessMetrics. Not only will this give us more up-to-date numbers, it will probably be less CPU too.